test/e2e/density.go

```diff
@@ -186,6 +209,108 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			if itArg.runLatencyTest {
+				Logf("Schedling additional Pods to measure startup latencies")
+				// Use service to assure even spread of pods between nodes
```

Comment:
I don't fully understand the need of the service. The scheduler should evenly spread pods across nodes. With multiple services the scheduler will, additionally, try to spread pods of a single service across nodes, but you only have 1 so it's the same as the first case?